cloudinit/dmi.py

```diff
@@ -120,6 +137,28 @@ def _read_kenv(key: str) -> Optional[str]:
     return None
 
 
+def _read_sysctl(key: str) -> Optional[str]:
+    """
+    Reads dmi data from OpenBSD's sysctl(8)
+    """
+    kmap = DMIDECODE_TO_KERNEL.get(key)
+    if kmap is None or kmap.openbsd is None:
+        return None
+
+    LOG.debug("querying dmi data %s", kmap.openbsd)
+
+    try:
+        cmd = ["sysctl", "-qn", kmap.openbsd]
+        (result, _err) = subp.subp(cmd)
```

Comment:
style nit: parens aren't required in assignment, named tuple attribute could be referenced also to avoid the unused assignment

```python
        result = subp.subp(cmd).stdout
```

not a required change, just a style suggestion

Reply:
@holmanb any idea why unittests are now failing after thischange?